tests/software/pairtools/flip/main.nf

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PAIRTOOLS_FLIP } from '../../../../software/pairtools/flip/main.nf' addParams( options: [:] )
+
+workflow test_pairtools_flip {
+
+    input = [ [ id:'test', single_end:false ], // meta map
+              file("https://raw.githubusercontent.com/open2c/pairtools/master/tests/data/mock.4flip.pairs", checkIfExists: true) ]
```

Comment:
We should probably add some of this test data to [nf-core/test-datasets](https://github.com/nf-core/test-datasets/tree/modules) seeing as you have needed to re-use it quite a lot. Ideally, relative to the SARS-CoV-2 genome. Will merge this for now but we can replace the paths and md5sums later.

Reply:
Should I upload the test data or you will do it? It will be great if we have the test data involved in the module test dataset.